src/spikeinterface/comparison/comparisontools.py

```diff
@@ -132,62 +132,6 @@ def compute_matching_matrix(
         num_units_train2,
         delta_frames,
     ):
-        """
```

Comment:
Why do we need to remove the docstring here? It is great that you added the other (my bad!) it here but is not like we are paying real state for docstrings here.

Reply:
Sorry, for me it was better for the higher function. This is why I moved it.
Having docstring here for function nested in another one will be never generated in the doc nor directly accessible. We can put it back maybe maybe not.

Reply:
That's a good point that I did not consider, but I have two points:
* It is still useful for the developer reading the code.
* I am taking your point that we should modify the way we are doing the numba functions here seriously. When we do that, I am expecting that we would be able to expose the docstrings again. And for that time, I would not like to have to urge the commit history to fetch it back. 

Given that is costing nothing, would you mind leaving it just as it is? We can discuss this after the release and I can myself go and create another PR to remove if you want it. But I don't want to get this discussion get in the way of the release and I don't want to lose the docstring to the commit history either.

Reply:
OK done.